### DISCUSSION TITLE
Debug no Safari 10 via windows

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#1565] [Discussão]
Ola pessoal , beleza?
Um cliente pediu compatibilidade do seu site no safari10.
Mas nem tenho ideia de como debugar isso tendo uma maquina windows.
Sei do BrowserStack,mas o cliente nao quer pagar.
Tentei maquina virtual via hackintosh, mas pra acessar o site do cliente e via vpn, e nao consegui fazer funcionar.
Alguem tem alguma sugestao?

**Comment by kivervinicius** [comment]
Em sampa não rola alugar um equipamento?

**Comment by rubenmarcus** [comment]
@kivervinicius , o cliente tinha conta no SauceLabs. já ajudou. Mas safari 10, pelo que sei é a versao default do Mac Os Sierra. Acho dificil achar um mac com essa versão de OS
